static/app/utils/theme/theme.tsx

```diff
@@ -252,6 +252,41 @@ const generateTokens = (colors: Colors) => ({
   },
 });
 
+const generateMotion = () => ({
+  duration: {
+    xs: '80ms',
+    sm: '120ms',
+    md: '160ms',
+    lg: '240ms',
+    xl: '320ms',
+    '2xl': '480ms',
+    '3xl': '640ms',
+  },
+  ease: {
+    enter: 'cubic-bezier(0.24, 1, 0.32, 1)',
+    exit: 'cubic-bezier(0.64, 0, 0.8, 0)',
+    default: 'cubic-bezier(0.72, 0, 0.16, 1)',
+    snap: 'cubic-bezier(0.8, -0.4, 0.5, 1)',
+  },
+  js: {
```

Comment:
I went down the path of generating these from the CSS values, but it's less obvious what's available when these values are dynamic, so I decided to keep these manually in sync for now.

Reply:
@natemoo-re I'm not a fan of the `js` object values. It's surface area for people to misuse them, and they'll likely just end up being used in css. 

Is the main use case here to use these in framer motion? If so, why don't we provide a motion animation object out of the box?

Reply:
Yep, `framer-motion` is the use case I had in mind for this, but I agree that a wrapper component would make more sense for that use case. Unfortunately we can't reuse the CSS tokens for that since it's a JS-first API.